ak_vendor/report/report.py

```diff
@@ -365,6 +372,10 @@ def create_asvs(cls, code: str, title: str) -> ASVS:
     def create_mstg(cls, code: str, title: str) -> MSTG:
         return MSTG(code=code, title=title)
 
+    @classmethod
+    def create_masvs(cls, code: str, title: str) -> MASVS:
```

Comment:
The type annotation should be quoted. This is not a requirement per say but it's a good practice. Change the line to `def create_masvs(cls, code: str, title: str) -> "MASVS":`

Reply:
Made the changes.